server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/CompactionStrategy.java

```diff
@@ -28,6 +28,12 @@
 /**
  * The interface for customizing major compactions.
  * <p>
+ * Important: Compaction configurations should be migrated to use only the new compaction plugins as
```

Comment:
I think one thing to know is that concurrent compactions for a tablet will not occur when a strategy is put in place.

Reply:
Other than that this seem reasonable.